pylint/checkers/base.py

```diff
@@ -1316,19 +1427,13 @@ def _find_name_group(self, node_type):
         return self._name_group.get(node_type, node_type)
 
     def _raise_name_warning(self, node, node_type, name, confidence):
-        type_label = _NAME_TYPES[node_type][1]
-
-        hint_name = node_type.replace('_', '-') + '-name-hint'
-        hint_part = '%r template' % hint_name
-
-        if self.config.include_naming_hint:
-            hint_rgx = getattr(self.config, hint_name.replace('-', '_'))
-            hint_part += ' (hint: %r)' % hint_rgx
+        type_label = HUMAN_READABLE_TYPES[node_type]
```

Comment:
Yes, I saw how it looks. Even if we show ``snake_case`` in the message, how an user will see what that means exactly without looking over the actual code, since these values are hardcoded in the code itself? I disagree about it being a workaround, especially since it's going to work for naming styles only